packages/recommendations-vdom/src/types/ProductRecord.ts

```diff
@@ -0,0 +1,7 @@
+export type ProductRecord<TObject> = TObject & {
```

Comment:
should this not be part of the -core package?

Reply:
Indeed—will move them.

Reply:
See 2fba5ac75b74a6d4ad6b591e153a5f79f203bfed.